Fix parse error in core.lua

ixtenu opened the pull request:
I'm seeing a parse error in core.lua:

```
core.lua:136: unexpected symbol near '/'
```

I don't know Lua, but this looks like a division operator that was accidentally repeated.

ixtenu committed:
Fix parse error in core.lua

Fixes Lua parse error:
core.lua:136: unexpected symbol near '/'

milhnl commented:
Thanks for the bug report and the PR!

That's the floor division operator. I was not aware that this does not work for all Lua versions. It seems you're running Lua 5.2. I think a correct fix would be something like [this](https://gitlab.com/muhq/vis-lspc/-/commit/3cf4097332fc45f42b15acaa2f7b399a7e86e64c). Can you test that and update the pull request? Otherwise I'll fix it myself, no worries. It's just that I like that you want to contribute :)

If you run into anything else, just let me know.

ixtenu committed:
Replace x//y with math.floor(x/y)

Lua 5.2 does not support the floor division (//) operator.  Commit
89b174718a85 changed the floor division to division, but the correct fix
would be to combine that with math.floor().

ixtenu commented:
> That's the floor division operator. I was not aware that this does not work for all Lua versions. It seems you're running Lua 5.2.

Thanks for the explanation.  Looking into it, I can confirm that you are correct.  I am using `vis` from nixpkgs 24.11, which is building with `lua` [as a dependency](https://github.com/NixOS/nixpkgs/blob/nixos-24.11/pkgs/by-name/vi/vis/package.nix#L18).  In nixpkgs, the `lua` alias resolves to Lua 5.2.4 (for [some reason](https://github.com/NixOS/nixpkgs/issues/323083)).  If I override the Lua version (e.g. as below), the original code with the `//` operator works just fine.

```nix
  environment.systemPackages = with pkgs; [
    (vis.override { lua = pkgs.lua5_4_compat; })
  ];
```

> I think a correct fix would be something like [this](https://gitlab.com/muhq/vis-lspc/-/commit/3cf4097332fc45f42b15acaa2f7b399a7e86e64c). Can you test that and update the pull request?

Done.  See the 2nd commit.  It works with `vis` built with Lua 5.2.4 or 5.4.7.

milhnl commented:
Thanks! Interesting indeed. Merged.
